pkg/pillar/types/locationconsts.go

```diff
@@ -118,6 +118,13 @@ const (
 
 	// ContainerdContentDir - path to containerd`s content store
 	ContainerdContentDir = ContainerdDir + "/io.containerd.content.v1.content"
+
+	// VtpmdCtrlSocket is UDS to ask vtpmd to launch SWTP instances for VMs
+	VtpmdCtrlSocket = "/run/swtpm/vtpmd.ctrl.sock"
+	// SwtpmCtrlSocketPath SWTPM per-vm socket path
+	SwtpmCtrlSocketPath = "/run/swtpm/%s.ctrl.sock"
```

Comment:
How about documenting that this is a template and what the argument is expected to be (app UUID?).
Same for the other template below.